clib/cSig.mli

```diff
@@ -48,12 +50,29 @@ end
     See OCaml documentation for more information. Operations which
     can't be efficiently implemented for HSets are moved to OSetS. *)
 
-module type SetS = sig
-  include USetS
+module type SetS =
+sig
```

Comment:
```suggestion
module type SetS = sig
```

Reply:
Should all such lines be formatted like that? I also forgot the reason, is it so that emacs can match `sig` and `end?

Reply:
I personally prefer the style where sig is on its own line.

Reply:
Can we say that is it a purely aesthetical issue or are there known associated technical consequences?